egs/sprakbanken_swe/s5/local/sprak_data_prep.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+# Copyright 2009-2012  Microsoft Corporation  Johns Hopkins University (Author: Daniel Povey)
+# Copyright 2013-2014  Mirsk Digital Aps (Author: Andreas Kirkedal)
+# Copyright 2016 KTH Royal Institute of Technology (Author: Emelie Kullmann)
+# Apache 2.0.
+
+
+dir=`pwd`/data/local/data
+lmdir=`pwd`/data/local/transcript_lm
+traindir=`pwd`/data/local/trainsrc
+testdir=`pwd`/data/local/testsrc
+rm -rf $lmdir $traindir $testdir $devdir
+mkdir -p $dir $lmdir $traindir $testdir $devdir
+local=`pwd`/local
+utils=`pwd`/utils
+
+
+. ./path.sh
+
+# Checks if python3 is available on the system and install python3 in userspace if not
+# This recipe currently relies on version 3 because python3 uses utf8 as internal 
+# string representation
+
+if ! which python3 >&/dev/null; then
+  echo "Python3 is not installed, to install it you should probably do:"
+  echo "sudo apt-get install python3" || exit 1;
+fi
+
+if [ ! -d $dir/download ]; then
+    mkdir -p $dir/download/0467-1 $dir/download/0467-2 $dir/download/0467-3
+fi 
+
+echo "Downloading and unpacking sprakbanken to $dir/corpus_processed. This will take a while."
+
+if [ ! -f $dir/download/sve.16khz.0467-1.tar.gz ]; then 
+    ( wget http://www.nb.no/sbfil/talegjenkjenning/16kHz/sve.16khz.0467-1.tar.gz --directory-prefix=$dir/download )
+fi
+
+if [ ! -f $dir/download/sve.16khz.0467-2.tar.gz ]; then 
+    ( wget http://www.nb.no/sbfil/talegjenkjenning/16kHz/sve.16khz.0467-2.tar.gz --directory-prefix=$dir/download )
+fi
+
+if [ ! -f $dir/download/sve.16khz.0467-3.tar.gz ]; then 
+    ( wget http://www.nb.no/sbfil/talegjenkjenning/16kHz/sve.16khz.0467-3.tar.gz --directory-prefix=$dir/download )
+fi
+
+if [ ! -f $dir/download/sve.16khz.0467-1.tar.gz ]; then 
+    ( wget http://www.nb.no/sbfil/talegjenkjenning/16kHz/sve.16khz.0468.tar.gz --directory-prefix=$dir/download )
+fi    
+
+echo "Corpus files downloaded."
+
+if [ ! -d $dir/download/0468 ]; then
+    echo "Unpacking files."
+    tar -xzf $dir/download/sve.16khz.0467-1.tar.gz -C $dir/download/0467-1
+    tar -xzf $dir/download/sve.16khz.0467-2.tar.gz -C $dir/download/0467-2
+    tar -xzf $dir/download/sve.16khz.0467-3.tar.gz -C $dir/download/0467-3
+    tar -xzf $dir/download/sve.16khz.0468.tar.gz -C $dir/download/0468    
+
+     
+    echo "Corpus unpacked succesfully."
+fi
+
+
+sph2pipe=$KALDI_ROOT/tools/sph2pipe_v2.5/sph2pipe
```

Comment:
I think this should be 
sph2pipe=$(which sph2pipe) || sph2pipe=$KALDI_ROOT/tools/sph2pipe_v2.5/sph2pipe